synapse/tests/test_lib_stormtypes.py

```diff
@@ -3465,10 +3465,16 @@ async def test_storm_lib_view(self):
             layrs0 = (layr0.iden, ldef1.get('iden'))
             layrs1 = (ldef1.get('iden'), layr0.iden)
 
+            # fork the default view to test editing the root view layers
+            fork00 = await core.callStorm('return($lib.view.get().fork().iden)')
+            self.eq(2, await core.callStorm('return($lib.view.get().layers.size())', opts={'view': fork00}))
+
             await core.callStorm('$lib.view.get().set(layers, $layers)', opts={'vars': {'layers': layrs0}})
             ldefs = await core.callStorm('return($lib.view.get().get(layers))')
             self.eq(layrs0, [x.get('iden') for x in ldefs])
 
+            self.eq(3, await core.callStorm('return($lib.view.get().layers.size())', opts={'view': fork00}))
+
```

Comment:
Check layer idens here to make sure the new fork has the expected layers and in the right order.